temporian/implementation/numpy/operators/propagate.py

```diff
@@ -0,0 +1,72 @@
+"""Implementation for the Propagate operator."""
+
+
+from typing import Dict
+
+import numpy as np
+import itertools
+
+from temporian.implementation.numpy.data.event import NumpyEvent, NumpyFeature
+from temporian.implementation.numpy.data.sampling import NumpySampling
+from temporian.core.operators.propagate import Propagate
+
+
+class PropagateNumpyImplementation:
+    """Numpy implementation for the propagat operator."""
+
+    def __init__(self, op: Propagate) -> None:
+        assert isinstance(op, Propagate)
+        self._op = op
+
+    def __call__(
+        self,
+        event: NumpyEvent,
+        add_event: NumpyEvent = None,

```

Comment:
Why the `None` default here? line 35 would raise if None

Reply:
Nice catch. This was some outdated code.

*Remove None